.github/workflows/integration-tests.yaml

```diff
@@ -96,3 +96,15 @@ jobs:
           python3 -m pip install -e sdk/python; pytest sdk/python/test --log-cli-level=info --namespace=default
         env:
           GANG_SCHEDULER_NAME: ${{ matrix.gang-scheduler-name }}
+
+      - name: Collect volcano logs
+        if: ${{ failure() &&  matrix.gang-scheduler-name == 'volcano' }}
```

Comment:
I couldn't see that this setting works fine.

https://github.com/tenzen-y/training-operator/actions/runs/5349550238/jobs/9700924693

Can you re-check this behavior and show an example actions job when `Deploy training operator` fail?

Reply:
Here is the snapshot in your given link.

<img width="858" alt="image" src="https://github.com/kubeflow/training-operator/assets/21003791/bf5c2851-001a-42dc-888a-8ac2cc47f079">

Reply:
FYI: https://github.com/lowang-bh/training-operator/actions/runs/5352956245/jobs/9708382734

Reply:
Thank you for checking and sharing this!